app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1129,6 +1129,12 @@ private void onBroadcastReceived(final Intent intent) {
                 // Close it because when changing orientation from portrait
                 // (in fullscreen mode) the size of queue layout can be larger than the screen size
                 closeItemsList();
+                // When the orientation changed, the screen height might be smaller.
+                // If the end screen thumbnail is not re-scaled,
+                // it can be larger than the current screen height
+                // and thus enlarging the whole player.
+                // This causes the seekbar to be ouf the visible area.
+                updateEndScreenThumbnail();
```

Comment:
Why not use javadoc above the method declaration instead of inline comments. Javadoc has IDE support.

Reply:
Done